tests/flask_integration/test_filters.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+from unittest.mock import Mock, patch
+
+from markupsafe import Markup
+
+from arbeitszeit_flask.filters import icon_filter
+from tests.flask_integration.flask import ViewTestCase
+
+
+class TestIconFilter(unittest.TestCase):
+    def setUp(self):
+        super().setUp()
+        self.mock_file_reader = Mock()
+
+    def tearDown(self):
+        super().tearDown()
+
+    def test_icon_with_default_attributes(self):
```

Comment:
In general we try to give our test functions more specific names, including for example a predicate/verb. I think the longer function names are justified by the better readability and by the fact that we do not call these test functions anywhere manually. It also forces us to think about what functionality we are testing exactly in each of these functions. Please let me know if you disagree.

Reply:
You are right 👍  I will improve the test case files. I also, will:

- remove unused `setUp` and `tearDown` functions
- double-check if more unit-tests are needed

Question. Are you OK with the decision that I mixed (Flask layer) unit tests of `icon_filter` with the integration test of the registered filter `icon` which is derived from `icon_filter`? Couldn't find a good place for the (Flask layer) unit tests in the `tests/*` folder.

Reply:
> Are you OK with the decision that I mixed (Flask layer) unit tests of `icon_filter` with the integration test

I wouldn't know, where to put it elsewhere, so I'm fine with it.